### PR TITLE
Portfolio: validate-contact-form 

### PR DESCRIPTION
**Pull request by silviatofana**
For this milestone of your portfolio website, you will implement a client-side validation of the data in the contact form before submission.
You need to implement a simple validation:
The content of the email field has to be in lower case.
You should implement the following interactions:
When the user submits the form, you check if the email is in lower case.
If the validation is OK, the form is sent.
If the validation is not OK, you show an error message to the user near the submit button informing them of the error, and the form is not sent.
When you create the error message, you should stick to the design guidelines (e.g., font, colors, margins) of [the templates in Figma].

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -39,7 +39,9 @@ <h1 class="logo">My Logo</h1>
     <section class="landing-page landing">
         <h1>I'm Silvia Glad to see you!</h1>
         <p class="prg">
-            I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            I'm a software developer! I can help you build a product ,
+             feature or website Look through some of my work and experience!
+              If you like what you see and have a project you need coded, don't hestiate to contact me.
```
Comment (KayLemba) [inline index.html:44]: 1. Please kindly add a background image in you header section. Follow design on Figma.


Your Design:
Screenshot from 2022-05-12 18-14-41

Figma Design:

Screenshot from 2022-05-12 18-16-14

2. In Your Cards section kindly add proper spacing between Canopy, backend dev and 2015 and make sure the dots are big enough. Follow the design on figma

This is Your design:
Screenshot from 2022-05-12 18-14-52

Follow this design on Figma:

Screenshot from 2022-05-12 18-16-19

3. In your about section please put the LET CONNECT text in its correct postion and also adjust the drop down and the side arrow on SKills. Check Design on Figma.
Your design:
Screenshot from 2022-05-12 18-15-10

Figma Template:
Screenshot from 2022-05-12 18-16-27

**Review by KayLemba — CHANGES_REQUESTED** [review]
Hi @silviatofana 👋
Good job so far working on the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉
No linter errors. ✔️
GitHub flow is followed. ✔️

Required Changes ♻️
Check the comments under the review in the inline comments.
- Please add a screenshot showcasing your UI  to read README file to make it look more proffessional
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (silviatofana) [reply index.html:44]: ok, thanks for all the helpful advice I will follow

**Review by ShahierNashaat — APPROVED** [review]
Hi @silviatofana , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by silviatofana** [comment]
thank you so much

**Pull request merged**